templates/openstack/features/index.html

```diff
@@ -0,0 +1,541 @@
+    {%- if slot == 'title' -%}
+      <h2>Way more than just a cloud</h2>
+    {%- endif -%}
+
+    {%- if slot == 'description' -%}
+      <p>
+        Rely on our expertise to give your enterprise everything it needs. When deployed by us Canonical OpenStack is way more than just a cloud. It is an end-to-end cloud solution.
+      </p>
+
+      <!-- Optional CTA block -->
+      <div class="p-cta-block u-no-padding--bottom">
+        <a href="/openstack/support" class="p-button--positive">Explore Canonical’s services</a>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h3 class="p-heading--5">Expandable containers layer</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>
+        Need a Kubernetes cluster? No problem: simply get it from the dashboard. Canonical OpenStack comes with an expandable containers layer, enabling your teams to provision <a href="https://ubuntu.com/kubernetes">Canonical Kubernetes</a> installations on demand.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h3 class="p-heading--5">Cloud storage</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>
+        Store up to 128 TB of data per node with no additional cost. <a href="https://ubuntu.com/ceph">Canonical Ceph</a>, included by default, is a highly scalable storage platform, providing both traditional and cloud-native storage options as required in heterogenous environments.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+      <h3 class="p-heading--5">Observability stack</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>
+        Equip your operations teams with an integrated <a href="/observability">observability stack</a>. Metrics, distributed traces, logs, and alerts: everything is collected automatically and unified under a single, intuitive dashboard.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_4' -%}
+      <h3 class="p-heading--5">Centralized system management</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_4' -%}
+      <p>
+        Automate security patching, auditing, and compliance tasks across your entire Ubuntu estate. <a href="https://ubuntu.com/landscape">Landscape</a> is a centralized system management tool that lets you enroll all those processes at scale.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_5' -%}
+      <h3 class="p-heading--5">Bare metal automation</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_5' -%}
+      <p>
+        Modernize your data center processes, such as server provisioning, with <a href="https://maas.io/">Canonical MAAS</a>. A full bare metal automation platform is included in the price of Canonical’s consulting packages for OpenStack.
```

Comment:
```suggestion
        Modernize your data center processes, such as server provisioning, with <a href="https://canonical.com/maas">Canonical MAAS</a>. A full bare metal automation platform is included in the price of Canonical’s consulting packages for OpenStack.
```
maas.io now redirects to canonical.com/maas